release-notes/opensearch-release-notes-1.3.11.md

```diff
@@ -0,0 +1,27 @@
+# OpenSearch and OpenSearch Dashboards 1.3.11 Release Notes
+
+## Release Details
+
+[OpenSearch and OpenSearch Dashboards 1.3.11](https://opensearch.org/versions/opensearch-1-3-11.html) includes the following bug fixes, infrastructure, enhancements and maintenance updates.
+
+OpenSearch [Release Notes](https://github.com/opensearch-project/OpenSearch/blob/main/release-notes/opensearch.release-notes-1.3.11.md).
+
+OpenSearch Dashboards [Release Notes](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/release-notes/opensearch-dashboards.release-notes-1.3.11.md).
+
+
+## ENHANCEMENTS
+
+## BUG FIXES
+
```

Comment:
Thanks @jordarlu please remove the enhancements and bug fixes headings as there are no entries.

Thanks.

Reply:
got it ... thanks .. removed